src/fscacher/tests/test_cache.py

```diff
@@ -342,3 +344,47 @@ def test_cache_control_envvar(
     c = PersistentCache(name="test-cache-control-envvar", envvar="MYCACHE_CONTROL")
     assert clear_spy.called is cleared
     assert c._ignore_cache is ignored
+
+
+@pytest.mark.skipif(shutil.which("git-annex") is None, reason="git annex required")
+def test_follow_moved_symlink(cache, tmp_path):
+    calls = []
```

Comment:
some test runs seems stuck in yellow...
I don't think we install `git-annex` in CI, we should, at least on linux and osx where we know that symlinks should generally be supported/available

Reply:
git-annex installed, but GitHub Actions seems to be having some queuing/runner availability problem that is keeping the tests from running.

Reply:
hm, https://www.githubstatus.com/ and https://www.githubstatus.com/history report nothing .  Another force push may be?

Reply:
although may be no need -- shows that this code is covered now and whatever test runs happened, are green